src/data/finset/lattice.lean

```diff
@@ -296,13 +296,13 @@ by simp [max']
 theorem min'_lt_max' {i j} (H1 : i ∈ s) (H2 : j ∈ s) (H3 : i ≠ j) : s.min' H < s.max' H :=
```

Comment:
This one also has redundant assumptions, and I guess it's not the only one around